src/uv-iiif-config.json

```
@@ -43,6 +43,11 @@
       "options": {
         "downloadEnabled": true
       }
+    },
+    "shareDialogue": {
+      "options": {
+        "embedTemplate": "<iframe src=\"{0}/assets/uv.html#?{1}\" width=\"{2}\" height=\"{3}\" allowfullscreen frameborder=\"0\" title=\"{4}\"></iframe>"
```

I don't think `assets` is supposed to be included here -- it results in invalid embed URLs like https://universalviewer-git-fork-crhallberg-embed-web-7151ad-bl-uv-team.vercel.app/assets/uv.html#?manifest=https%3A%2F%2Fwellcomelibrary.org%2Fiiif%2Fb18035723%2Fmanifest&c=0&m=0&cv=0&config=&locales=en-GB%3AEnglish+%28GB%29%2Ccy-GB%3ACymraeg%2Cfr-FR%3AFran%C3%A7ais+%28FR%29%2Cpl-PL%3APolski%2Csv-SE%3ASvenska&xywh=-2186%2C-396%2C6940%2C4334&r=0

```suggestion
        "embedTemplate": "<iframe src=\"{0}/uv.html#?{1}\" width=\"{2}\" height=\"{3}\" allowfullscreen frameborder=\"0\" title=\"{4}\"></iframe>"
```

---

Note that if you take my suggestion below, this fix is not really relevant... but I figured I'd include it anyway since it is a bug. :-)